File: Benchmarks/enterprise/java/jakarta/testcode/BenchmarkTest13534.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import jakarta.ws.rs.*;
import jakarta.ws.rs.core.Context;
import jakarta.ws.rs.core.MediaType;
import jakarta.ws.rs.core.Response;

@Path("/")
public class BenchmarkTest13534 {

    private static String trimEnds(String v) { return v.trim(); }

    @GET
    @Path("/BenchmarkTest13534")
    @Produces(MediaType.APPLICATION_JSON)
    public Response BenchmarkTest13534(@HeaderParam("Origin") String origin, @Context HttpServletRequest request, @Context HttpServletResponse response) throws Exception {
        String originValue = origin != null ? origin : "";
        String data = trimEnds(originValue);
        return Response.ok("{\"resource\":\"" + data + "\"}", MediaType.APPLICATION_JSON).build();
    }
}

File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_13534.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request
import requests
from starlette.responses import HTMLResponse


async def BenchmarkTest13534(request: Request):
    user_id = request.query_params.get('id', '')
    data = '%s' % (user_id,)
    return HTMLResponse('<script src="' + str(data) + '"></script>')
